docker/php.Dockerfile

```diff
@@ -159,23 +160,26 @@ ARG VERSION=latest
 ENV VERSION="${VERSION}"
 LABEL maintainer="Flavio Heleno <flaviohbatista@gmail.com>" \
       org.opencontainers.image.authors="flaviohbatista@gmail.com" \
+      org.opencontainers.image.base.name="ghcr.io/package-health/pph-php-cli:${VERSION}" \
+      org.opencontainers.image.source="https://github.com/package-health/php" \
       org.opencontainers.image.title="PHP-Package-Health: PHP-CLI" \
       org.opencontainers.image.url="https://github.com/package-health/php" \
       org.opencontainers.image.vendor="Package Health" \
-      org.opencontainers.image.version="${VERSION}" \
-      org.opencontainers.image.base.name="ghcr.io/package-health/pph-php-cli:${VERSION}"
+      org.opencontainers.image.version="${VERSION}"
+
+VOLUME ["/var/www/html/run"]
 
 ENTRYPOINT ["/usr/bin/dumb-init", "--"]
 CMD ["php"]
 
 #============================================
 # FPM SAPI
 #============================================
-FROM php:8.1.6-fpm-alpine3.16 as fpm
+FROM php:8.1.12-fpm-alpine3.16 as fpm
 
 # https://blog.packagecloud.io/eng/2017/02/21/set-environment-variable-save-thousands-of-system-calls/
-ENV TZ=:/etc/localtime
-ENV PHP_ENV=development
+ENV TZ=:UTC
+ENV PHP_ENV=dev
```

Comment:
@okiedork should this be kept as "development"?

Reply:
`dev`.